<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="51cc9560-a0fb-486d-bb48-8b8d283a6c3e" name="ChetanBadhe@CHETANBADHE-PC 2014-09-16 22:48:33" runUser="ChetanBadhe-PC\ChetanBadhe" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="15fb7c02-8048-47b3-ab4d-8203306c1aa3">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="ChetanBadhe_CHETANBADHE-PC 2014-09-16 22_48_33">
      <DeploymentItem filename="C:\Users\ChetanBadhe.ChetanBadhe-PC\Desktop\KTLDemo\KTLDemo\bin\KTLDemo.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2014-09-16T22:48:33.0811233-07:00" queuing="2014-09-16T22:48:35.6012675-07:00" start="2014-09-16T22:48:36.0392925-07:00" finish="2014-09-16T22:48:38.0364068-07:00" />
  <ResultSummary outcome="Failed">
    <Counters total="1" executed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" passed="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="CreateDataContext_ConnectionString_ReturnsDataContextObject" storage="c:\users\chetanbadhe.chetanbadhe-pc\desktop\ktldemo\testproject1\bin\debug\testproject1.dll" id="452ff378-94b3-5a6a-6cdb-ebb55fc590e6">
      <Execution id="5e7461fb-8634-4b6d-850d-64909f8c097e" />
      <TestMethod codeBase="C:/Users/ChetanBadhe.ChetanBadhe-PC/Desktop/KTLDemo/TestProject1/bin/Debug/TestProject1.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="TestProject1._DefaultTest, TestProject1, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="CreateDataContext_ConnectionString_ReturnsDataContextObject" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="452ff378-94b3-5a6a-6cdb-ebb55fc590e6" executionId="5e7461fb-8634-4b6d-850d-64909f8c097e" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="5e7461fb-8634-4b6d-850d-64909f8c097e" testId="452ff378-94b3-5a6a-6cdb-ebb55fc590e6" testName="CreateDataContext_ConnectionString_ReturnsDataContextObject" computerName="CHETANBADHE-PC" duration="00:00:00.5155462" startTime="2014-09-16T22:48:36.1813007-07:00" endTime="2014-09-16T22:48:37.8803978-07:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="5e7461fb-8634-4b6d-850d-64909f8c097e">
      <Output>
        <ErrorInfo>
          <Message>Test method TestProject1._DefaultTest.CreateDataContext_ConnectionString_ReturnsDataContextObject threw exception: 
System.NullReferenceException: Object reference not set to an instance of an object.</Message>
          <StackTrace>    at KTLDemo.Model.DCOrdersDataContext..ctor() in C:\Users\ChetanBadhe.ChetanBadhe-PC\Desktop\KTLDemo\KTLDemo\Model\DCOrders.designer.cs:line 47
   at TestProject1._DefaultTest.CreateDataContext_ConnectionString_ReturnsDataContextObject() in C:\Users\ChetanBadhe.ChetanBadhe-PC\Desktop\KTLDemo\TestProject1\_DefaultTest.cs:line 81
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
</TestRun>